app/Console/Commands/ImportAshesCampaigns.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Carbon\Carbon;
+use League\Csv\Reader;
+use Rogue\Models\Campaign;
+use Rogue\Services\Registrar;
+use Illuminate\Console\Command;
+
+class ImportAshesCampaigns extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:ashescampaignsimport {path}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Imports legacy campaigns from Ashes based on the data in the given (link to a) CSV file.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(Registrar $registrar)
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        // Make a local copy of the CSV
+        $path = $this->argument('path');
+        info('rogue:legacycampaignimport: Loading in csv from ' . $path);
```

Comment:
@DFurnes not sure what you mean by that?

Reply:
@chloealee that would be doing `$this->line('rogue:legacycampaignimport: Loading in csv from ' . $path)`. It'll print out in your terminal locally and when running on a one-off dyno in Heroku, it will print to the logs for that dyno (which is [something we figured out semi-recently](https://devcenter.heroku.com/articles/one-off-dynos#running-tasks-in-background)!)

Reply:
I believe it will still log to PaperTrail

Reply:
ooooo what are the benefits of logging this way?